answers/AutoloadingClasses/dc50b2e4-32be-45e3-948f-0a0e9192afea/includes/autoload.php

```diff
@@ -0,0 +1,12 @@
+<?php
+function myAutoloader($className)
+{
+    $file = dirname(__DRI__) . '\\src\\' . $className . '.php';
```

Comment:
`__DRI__` typo?

Reply:
My mistakes... will edit this... Thank you.